pwiz_tools/Skyline/Model/Lib/SpectrumRanker.cs

```diff
@@ -0,0 +1,835 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using pwiz.Common.Chemistry;
+using pwiz.Common.Collections;
+using pwiz.Common.SystemUtil;
+using pwiz.Skyline.Model.DocSettings;
+using RankedMI = pwiz.Skyline.Model.Lib.LibraryRankedSpectrumInfo.RankedMI;
+using pwiz.Skyline.Util;
+
+namespace pwiz.Skyline.Model.Lib
+{
```

Comment:
As you further hone your sense of just what it is this code does, it would be great to have a comment here describing that/